### DISCUSSION TITLE
How to configure webpack for create-react-app

### DISCUSSION THREAD

**Discussion by mmehmoodahmed** [webpack/webpack#15858] [General]
I want to configure webpack for create-react-app .
I am using sass and other packages as well.

Is there any way to create  the plugin plain names without hashing ?

**Comment by vankop** [comment]
please ask in create react app issue tracker

**Reply by mmehmoodahmed** [reply]
Who 'll reply to it. I am waiting. 

https://github.com/facebook/create-react-app/issues/12434
